app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,52 @@
+class CommentsController < ApplicationController
+  load_and_authorize_resource
+
+  # issue_comments POST /issues/:issue_id/comments(.:format) comments#create
+  def create
+    @resource = Issue.find(params[:issue_id])
+    authorize! :read, @resource
```

Comment:
No need for this here

Reply:
If this is not done, by changing the url from inspector tools the comment can be posted in even other company from this company

Reply:
resource in now being loaded via cancan